tests/monotouch-test/SystemConfiguration/CaptiveNetworkTest.cs

```diff
@@ -173,7 +173,7 @@ public void SetSupportedSSIDs ()
 			bool supported = true;
 #else
 			// that API is deprecated in iOS9 - and it might be why it returns false (or not)
-			bool supported = !TestRuntime.CheckXcodeVersion (7, 0);
+			bool supported = false;
```

Comment:
I wonder if that would work on old devices... because (last time I checked) this executed without failure